lib/core/datatable/components/datatable/datatable.component.html

```diff
@@ -197,7 +197,8 @@
                         </ng-container>
                     </div>
                     <div *ngIf="col.template" class="adf-datatable-cell-container">
-                        <div class="adf-cell-value" [attr.tabindex]="col.focus ? 0 : null">
+                        <!--ACA-2566 added check for data in the cell, and if there is add tabindex of 0 to it. if not, nothing blank will get focus -->
+                        <div class="adf-cell-value" [attr.tabindex]="data.getValue(row, col)? 0 :null">
```

Comment:
For custom templates, data column focus can be configured via `focus` property. In this way we give control of the element that will receive focus, depending on the content of the custom template.